webapp/src/main/java/life/qbic/views/login/LoginLayout.java

```diff
@@ -57,18 +53,11 @@ public LoginLayout(@Autowired LoginHandlerInterface loginHandlerInterface) {
 
   private void initLayout() {
     contentLayout = new VerticalLayout();
-    this.loginForm = new ConfigurableLoginForm();
-    loginForm.setAction("login");
```

Comment:
I think we need that. I cant recall but I think it was for the css file. Why did you remove it?

Reply:
Ahh found it nvm!